amundsen_application/static/js/components/Tags/TagInput/index.tsx

```diff
@@ -298,6 +307,7 @@ class TagInput extends React.Component<TagInputProps, TagInputState> {
           isValidNewOption={this.isValidNewOption}
           name="tags"
           noOptionsMessage={this.noOptionsMessage}
+          onClick={this.stopPropagation}
```

Comment:
The reason this doesn't work is that `CreateableSelect` is a 3rd party component, created just like all the custom components we create here, where it can only be interacted it via its properties. This component does not have an `onClick` method. See the link left in the comments: https://react-select.com/props

However you should be able to put an `onClick={this.doTheThing}` on the parent `<div>` in this file for it to get triggered.